sdk/eventhub/event-hubs/review/event-hubs.api.md

```diff
@@ -22,6 +22,12 @@ import { TokenCredential } from '@azure/core-amqp';
 import { TokenType } from '@azure/core-amqp';
 import { WebSocketImpl } from 'rhea-promise';
 
+// @public
+export interface BatchOptions {
+    maxMessageSize?: number | null;
+    partitionKey?: string | null;
```

Comment:
Does the types here need to have the `| null`? Can we do without it?

Reply:
Yes we can. I used the same as `SendOptions`